src/app/app.module.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2017 VMware, Inc. All Rights Reserved.
```

Comment:
Re-reading this it might be better to handle all the copyright dates at once. Maybe a coordinated PR in early January could take care of all of the files?

Reply:
Yeah that's a good point. I think we have a mixed variety and my regex rule accepts both (2017 or 2016-2017). Can we create a separate task for this?

Reply:
Yes, thats what I was thinking. 
I'll open an issue for it.